tests/models/embedding/language/test_truncation_control.py

```diff
@@ -0,0 +1,120 @@
+# SPDX-License-Identifier: Apache-2.0
+import pytest
+
+from vllm.entrypoints.llm import LLM
+
+llm = LLM(model="intfloat/multilingual-e5-large", task="embed")
+
+small_input_str = "In which state the capital of the United States is located?"
+
+big_input_str = "Immerse yourself in the enchanting \
```

Comment:
You can could a `conftest.py` file to add this big string in a fixture so that it doesn't have to be redefined twice.